pages/api/team.js

```diff
@@ -0,0 +1,90 @@
+import AirtablePlus from 'airtable-plus'
+
+const airtable = new AirtablePlus({
+    baseID: 'app79wD9AFys1NMUp',
+    apiKey: process.env.AIRTABLE_API_KEY,
+    tableName: 'Current',
+})
+
+const cache = {
+    refreshed_at: 0,
+    current: [],
+    acknowledged: []
+};
+
+export async function fetchTeam() {
+    if ((cache.refreshed_at + (5 * 60 * 1000)) >= Date.now()) {
+        return cache
+    }
+
+    const records = await airtable.read();
+    const current = [];
+    const acknowledged = [];
+
+    for (let record of records.sort((a, b) => a.fields['Order'] - b.fields['Order'])) {
+        const member = {
+            name: record.fields["Name"],
+            bio: record.fields["Bio"] || null,
```

Comment:
use null coalescing instead

Reply:
Null coalescing wouldn't work as if any of the values are undefined then it crashes on build with an error like:

(I'm in the process of changing to use `getStaticProps` instead of `getServerSideProps`)

```
Error: Error serializing `.team.current[1].bio` returned from `getStaticProps` in "/team".
Reason: `undefined` cannot be serialized as JSON. Please use `null` or omit this value.
```